src/js/components/tour/landing.tsx

```diff
@@ -0,0 +1,120 @@
+import Button from '@salesforce/design-system-react/components/button';
+import Card from '@salesforce/design-system-react/components/card';
+import Modal from '@salesforce/design-system-react/components/modal';
+import i18n from 'i18next';
+import React from 'react';
+import { Trans } from 'react-i18next';
+
+import backpackSvg from '!raw-loader!~img/backpack-lg.svg';
+import mapSvg from '!raw-loader!~img/map-lg.svg';
+import seesawSvg from '!raw-loader!~img/seesaw-lg.svg';
+import { Illustration } from '~js/components/utils';
+import { WALKTHROUGH_TYPES, WalkthroughType } from '~js/utils/constants';
+
+export interface Tour {
+  header: string;
+  tag: string;
+  linkText: string;
+  type: WalkthroughType;
+  icon: string;
+  disabled?: boolean;
+}
+
+const LandingModal = ({
+  isOpen,
+  runTour,
+  onRequestClose,
+}: {
+  isOpen: boolean;
+  runTour: (type: WalkthroughType) => void;
+  onRequestClose: () => void;
+}) => {
+  const tours: Tour[] = [
+    {
+      header: i18n.t('I want to Play'),
+      tag: i18n.t('Make a Scratch Org to view Project & play.'),
+      linkText: i18n.t('Start Play Walkthrough'),
+      type: WALKTHROUGH_TYPES.PLAY,
+      icon: seesawSvg,
+      disabled: true,
+    },
+    {
+      header: i18n.t('I want to Help'),
+      tag: i18n.t('Browse available Tasks; give your input.'),
+      linkText: i18n.t('Start Help Walkthrough'),
+      type: WALKTHROUGH_TYPES.HELP,
+      icon: backpackSvg,
+      disabled: true,
+    },
+    {
+      header: i18n.t('I want to Plan'),
+      tag: i18n.t('Create a Task or an Epic; add your work.'),
+      linkText: i18n.t('Start Plan Walkthrough'),
+      type: WALKTHROUGH_TYPES.PLAN,
+      icon: mapSvg,
+    },
+  ];
+
+  return (
+    <Modal
```

Comment:
Dialogs should have labels: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#dialogs_should_be_labeled

(I saw that warning when inspecting the joyride dialogs, but assume it applies here as well - and I don't see where to note it in the joyride code)

Reply:
@mirisuzanne We don't really control this markup, but I'm seeing both `aria-label` and `aria-describedby`. Is there something else missing?

![Screen Shot 2021-03-29 at 10 49 00 AM](https://user-images.githubusercontent.com/552316/112855079-7c425b00-907c-11eb-9c3e-7fd8d8515ab3.png)